addOns/domxss/CHANGELOG.md

```diff
@@ -15,6 +15,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Maintenance changes.
 - Promote to beta
 
+## [10] - 2020-12-01
+### Changed
+- Now clicking on different buttons throughout the page to see if it triggers XSS.
```

Comment:
This should be added to the above "Changed" section.

Reply:
> Can you explain how it avoiding to click on cancel buttons?
> Can you explain how it avoiding to click on cancel buttons?

The main reason I wanted to add this is so the plugin will click the cancel buttons. We have JavaScript similar to the below on one of our sites. When the Cancel button is clicked, the JavaScript that's in the retUrl attribute is triggered on the user's page. 
```
            $("#DIAGLOG").dialog("option", "buttons", {
                "Cancel": function () {
                    window.location = '@retUrl';
                },
                "Submit": function () {
                    xxxxxxxx
                    });
                }
            });
```

Reply:
> Can we tell they are buttons and not sendKeys to them?
What's an example of a sendKey?